src/Dfe.PlanTech.Domain/Content/Models/ComponentDropDown.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace Dfe.PlanTech.Domain.Content.Models
+{
+	public class ComponentDropDown : ContentComponent
```

Comment:
poor formatting here - use a tool like prettify or something to keep formatting correct

Reply:
fix this then check push - I'll approve it now